### ISSUE TITLE
interleaving the input images

### ISSUE THREAD

**Issue by sesmae** [nayeemrizve/OpenLDN#3]
Thank you for open-sourcing your code. Could you please explain what is the purpose of interleave/deinterleave functions used in lines 206 and 218 of [train-base.py](https://github.com/nayeemrizve/OpenLDN/blob/main/base/train-base.py)? Am I missing something?

**Comment by nayeemrizve** [comment]
Thank you for expressing interest in our work. The interleave/deinterleave functions are artifacts from the Pytorch version of the MixMatch code I was using. In that case, it is useful to get the correct estimate of batchnorm since the labeled and unlabeled data are passed separately. However, in this case, since all the labeled and unlabeled data are processed together, these functions are not basically doing anything. So, ideally, they can be removed. Hope this clarifies your concerns.
